helm/adelphi/tests/kind-values-trunk.yaml

```diff
@@ -0,0 +1,15 @@
+############################ 
+# Config overrides for KIND
+# Used in integration tests
+# Runs against C* trunk 
+############################
+
+storageClassName: standard
+
+clusterSize: 1
```

Comment:
This is the values file used by the integration tests for running on Kind in GithubActions.
We run with a single node on each cluster to avoid hitting the env limits.